src/gui/plugins/scene3d/GzScene3D.qml

```diff
@@ -98,4 +99,28 @@ Rectangle {
       GzScene3D.OnDropped(drop.text, drag.x, drag.y)
     }
   }
+
+  // pop error
+  Connections {
+    target: GzScene3D
+    onPopupError: errorPopup.open()
+  }
+
+  Dialog {
+    id: errorPopup
+    parent: ApplicationWindow.overlay
+    modal: true
+    focus: true
+    anchors.centerIn: parent
```

Comment:
This is failing for me on Bionic:

```
[GUI] [Err] [Plugin.cc:147] Failed to instantiate QML file [:/GzScene3D/GzScene3D.qml].
* file::/GzScene3D/GzScene3D.qml:114:5: Cannot assign to non-existent property "anchors"
```

I've run into this before :confused: 

https://github.com/ignitionrobotics/ign-gazebo/blob/9fbb9b988c20711fcff706fb14d034516e027e6b/src/gui/plugins/joint_position_controller/JointPositionController.qml#L98-L99

Reply:
Should be fixed 2d6a9301443ab06d3fb217ee22be9254781389b6